iclaims_test.go

```diff
@@ -174,11 +174,11 @@ func Test_IClaims_SetBootSeed_invalid(t *testing.T) {
 func Test_IClaims_SetCertificationReference_invalid(t *testing.T) {
 	tv := makeIClaims(t)
 
-	expectedErr := `wrong syntax for claim: MUST be in EAN-13 format`
+	expectedErr := `wrong syntax for claim: MUST be in EAN-13`
```

Comment:
Little bewildered how the dropped word 'format' is now picked as the set method error should return the string which ends with EAN-13 format?

Reply:
Apologies, I am not sure I understand this comment.

Here I am chopping the string at the point where P1 and P2 error messages diverge, so that I can have one assert for both cases.